puppeteer/node-puppeteer-prune.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+
+# This is a helper script which deletes some not needed files from NPM packages.
+# This script is specific for Puppeteer installations.
+
+MODULES_PATH="${1:-./node_modules}"
+
+# delete Puppeteer CommonJS modules, we use the ES modules (in lib/esm)
+rm -rf "$MODULES_PATH/puppeteer-core/lib/cjs"
```

Comment:
I kind of expect that it will run internally that node-prune.sh..so just one command will be enough to cleanup pupeteer modules.

Reply:
Both commands are started in the `agama-integration-test` scripts which installs the NPM packages and runs the tests. Normally you should not need to run them separately.